platform/src/test/java/platformstepdefinition/PF_Order.java

```diff
@@ -36,44 +49,104 @@ public void select_1st_kanji_product_from_product_list_page() throws Throwable {
-	public void step2_payment_with_valid_credit_card() throws Throwable {
-		orderObj.methodForStep2PaymentWithValidCreditCard();
+	@And("^Open Product List Page$")
+	public void open_product_list_page() throws Throwable {
+		orderObj.openProductListPage();
 	}
 
-	@And("^Step2 Payment With In Valid Credit Card$")
-	public void step2_payment_with_in_valid_credit_card() throws Throwable {
-		orderObj.methodForStep2PaymentWithInValidCreditCard();
+	@And("^Click on back button from order flow$")
+	public void click_on_back_button_from_order_flow() throws Throwable {
+		orderObj.clickOnOrderFlowBackButton();
+	}
+
+	@And("^Open MemberShip Plan Page from Top page$")
+	public void open_membership_plan_page_from_top_page() throws Throwable {
+		orderObj.openMemberPlanPageFromTopPage();
+	}
+
+	@And("^Apply MemberShip Plan from Top page$")
+	public void apply_membership_plan_from_top_page() throws Throwable {
+		orderObj.clickApplyForPrimeMembershipButton();
+	}
+
+	@And("^Confirm Order Flow Step 1 Page Load Successfully$")
+	public void confirm_order_flow_step_1_page_load_successfully() throws Throwable {
+		orderObj.orderFlowStep1PageLoadedSuccessfully();
+	}
+
+	@And("^Confirm Order Flow Step 2 Page Load Successfully$")
+	public void confirm_order_flow_step_2_page_load_successfully() throws Throwable {
+		orderObj.orderFlowStep2PageLoadedSuccessfully();
+	}
+
+	@And("^Confirm Order Flow Step 3 Page Load Successfully$")
+	public void confirm_order_flow_step_3_page_load_successfully() throws Throwable {
+		orderObj.orderFlowStep3PageLoadedSuccessfully();
+	}
+
+	@And("^Confirm Order Flow Step 4 thank you Page Load Successfully$")
+	public void confirm_order_flow_step_4_thank_you_page_load_successfully() throws Throwable {
+		orderObj.orderFlowStep4ThankYouPageLoadedSuccessfully();
+	}
+
+	@And("^Proceeed With 1st Casec Product$")
+	public void proceeed_with_1st_casec_product() throws Throwable {
+		orderObj.proceedWith1stCasecProduct();
+	}
+
+	@And("^Reglogin between Step 1 page to Step 2 page$")
+	public void reglogin_between_step_1_page_to_step_2_page() throws Throwable {
+		orderObj.methodForReloginForOrderFlow();
 	}
 
-	@And("^Step2 Payment With Blank Credit Card$")
+	@And("^Step 2 Payment With Valid Credit Card$")
+	public void step2_payment_with_valid_credit_card() throws Throwable {
+		orderObj.methodForStep2PaymentWithValidCreditCard();
+	}
+
+	@And("^Step 2 Payment With Blank Credit Card$")
 	public void step2_payment_with_blank_credit_card() throws Throwable {
 		orderObj.methodForStep2PaymentWithBlankCreditCard();
 	}
 
-	@And("^Step2 Payment With Valid Convenience Store$")
+	@And("^Step 2 Payment With InValid Credit Card$")
+	public void step_2_payment_with_invalid_credit_card() throws Throwable {
+		orderObj.methodForStep2PaymentWithInValidCreditCard();
+	}
+
+	@And("^Step 2 Payment With Valid Convenience Store$")
 	public void step2_payment_with_valid_convenience_store() throws Throwable {
 		orderObj.methodForStep2PaymentWithConStore();
```

Comment:
Please start method name with a verb
Cant determine what this method does
Please make changes wherever applicable